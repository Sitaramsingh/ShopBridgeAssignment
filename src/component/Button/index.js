import React from 'react';
// import styles from './button.css';
import PropTypes from 'prop-types';


function button(props) {
    return (
        <button className={['butn', props.className].join(' ')}
            onClick={props.onClick}
            disabled={props.disable}
        >{props.children}</button>
    );
}

button.propTypes = {
    className: PropTypes.string,
    onClick: PropTypes.func,
    disable: PropTypes.bool,
    children: PropTypes.any
};

export default button;