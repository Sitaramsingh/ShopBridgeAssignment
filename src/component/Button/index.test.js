import React from 'react';
import { shallow } from 'enzyme';
import Button from './index';


describe('Component should be render ', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Button/>);
    expect(component).toMatchSnapshot();
  });
});