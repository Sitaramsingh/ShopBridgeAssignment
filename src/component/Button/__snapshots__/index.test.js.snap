// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should be render  should render correctly in "debug" mode 1`] = `
<button
  className="butn "
/>
`;
